app/views/cheeses/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<ul>
+  <% @cheeses.each do |cheese| %>
+    <li>
+      <div> <%= link_to cheese.name, cheese %> </div>
```

Comment:
This will actually add a space before and after the text of your link, which might affect your styling